contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.4.0--5.0.0.sql

```diff
@@ -187,12 +187,35 @@ UPDATE sys.babelfish_authid_login_ext SET is_fixed_role = 1 WHERE rolname = 'sys
 
 -- At this point, there will be only one database fixed role which is db_owner.
 UPDATE sys.babelfish_authid_user_ext SET is_fixed_role = 1 WHERE orig_username = 'db_owner';
-UPDATE sys.babelfish_authid_user_ext SET is_fixed_role = 0 WHERE orig_username != 'db_owner';
 
 CREATE OR REPLACE PROCEDURE sys.create_db_roles_during_upgrade()
 LANGUAGE C
 AS 'babelfishpg_tsql', 'create_db_roles_during_upgrade';
-CALL sys.create_db_roles_during_upgrade();
+
+DO $$
+DECLARE
+    role_info record;
+BEGIN
+    WITH fixed_db_roles AS (
+        SELECT DISTINCT orig_username
+        FROM sys.babelfish_authid_user_ext
+        WHERE is_fixed_role = 1
+          AND orig_username IN ('db_owner', 'db_securityadmin', 'db_accessadmin', 
```

Comment:
let us remove 'db_owner' from this list otherwise the error message would be misleading.

Reply:
warning message will only come when other roles will exist where the it will show db_owner also exists along with other roles which is valid.